monkey/infection_monkey/monkey.py

```diff
@@ -86,6 +89,9 @@
 logger = logging.getLogger(__name__)
 logging.getLogger("urllib3").setLevel(logging.INFO)
 
+TMP_DIR = Path(gettempdir()) / random_filename()
```

Comment:
We'll need to actually create this directory at some point. Preferably in `_build_puppet()`